src/corelibs/U2Lang/src/support/WorkflowRunTask.cpp

```diff
@@ -339,7 +342,7 @@ Task::ReportResult WorkflowIterationRunTask::report() {
 }
 
 WorkerState WorkflowIterationRunTask::getState(const ActorId& id) {
-    if (scheduler) {
+    if (scheduler && debugInfo) {
```

Comment:
Don't you need to move debug info into the inner check?
```
debugInfo && debugInfo->isPaused()
```

Reply:
👍